docs/guides/quick-start.md

```diff
@@ -62,9 +63,9 @@ frictionless validate --help
 frictionless transform --help
 ```
 
-## Short Example
+## Example
 
-> For a more indepth example, use the [Overview Example](overview-example)
+> For more examples, use the [Basic Examples](basic-examples)
```

Comment:
I think those examples are also very basic so I would rather keep like one example vs other examples difference instead of simple vs indepth